card/spec/lib/card/name_spec.rb

```diff
@@ -1,158 +1,155 @@
     end
 
     it "should not singularize double s's" do
-      expect("grass".to_name.key).to eq('grass')
+      expect('grass'.to_name.key).to eq('grass')
     end
 
     it "should not singularize letter 'S'" do
       expect('S'.to_name.key).to eq('s')
     end
 
-    it "should handle unicode characters" do
-      expect("Mañana".to_name.key).to eq('mañana')
+    it 'should handle unicode characters' do
+      expect('Mañana'.to_name.key).to eq('mañana')
     end
 
-    it "should handle weird initial characters" do
+    it 'should handle weird initial characters' do
       expect('__you motha @#$'.to_name.key).to eq('you_motha')
       expect('?!_you motha @#$'.to_name.key).to eq('you_motha')
     end
 
-    it "should allow numbers" do
-      expect("3way".to_name.key).to eq('3way')
+    it 'should allow numbers' do
+      expect('3way'.to_name.key).to eq('3way')
     end
 
-    it "internal plurals" do
-      expect("cards hooks label foos".to_name.key).to eq('card_hook_label_foo')
+    it 'internal plurals' do
+      expect('cards hooks label foos'.to_name.key).to eq('card_hook_label_foo')
     end
 
-    it "should handle html entities" do
+    it 'should handle html entities' do
       # This no longer takes off the s, is singularize broken now?
-      expect("Jean-fran&ccedil;ois Noubel".to_name.key).to eq('jean_françoi_noubel')
+      expect('Jean-fran&ccedil;ois Noubel'.to_name.key).to(
+        eq('jean_françoi_noubel')
+      )
     end
   end
 
-
-
-
-
-  describe "#url_key" do
-    cardnames = ["GrassCommons.org", 'Oh you @##', "Alice's Restaurant!", "PB &amp; J", "Mañana"].map(&:to_name)
+  describe '#url_key' do
+    cardnames = ['GrassCommons.org', 'Oh you @##', "Alice's Restaurant!", 'PB &amp; J', 'Mañana'].map(&:to_name)
 
     cardnames.each do |cardname|
-      it "should have the same key as the name" do
-        k, k2 = cardname.key, cardname.url_key
-        #warn "cn tok #{cardname.inspect}, #{k.inspect}, #{k2.inspect}"
+      it 'should have the same key as the name' do
+        k = cardname.key
+        k2 = cardname.url_key
+        # warn "cn tok #{cardname.inspect}, #{k.inspect}, #{k2.inspect}"
         expect(k).to eq(k2.to_name.key)
       end
     end
   end
 
-  describe "#valid" do
-    it "accepts valid names" do
-      expect("this+THAT".to_name).to be_valid
-      expect("THE*ONE*AND$!ONLY".to_name).to be_valid
+  describe '#valid' do
+    it 'accepts valid names' do
+      expect('this+THAT'.to_name).to be_valid
+      expect('THE*ONE*AND$!ONLY'.to_name).to be_valid
     end
 
-    it "rejects invalid names" do
-#      "Tes~sd".to_name.should_not be_valid
-      expect("TEST/DDER".to_name).not_to be_valid
+    it 'rejects invalid names' do
+      # 'Tes~sd'.to_name.should_not be_valid
+      expect('TEST/DDER'.to_name).not_to be_valid
     end
 
-    it "rejects long names" do
+    it 'rejects long names' do
       card = Card.new
-      card.name="1"*256
+      card.name='1'*256
```

Comment:
Surrounding space missing for operator `=`.<br>Surrounding space missing for operator `*`.